resources/views/account/partials/buttons.blade.php

```diff
@@ -0,0 +1,8 @@
+<div class="flex flex-wrap w-full justify-between gap-2">
+    <x-rapidez-mw::button.outline href="/account">
+        @lang('Back to dashboard')
+    </x-rapidez-mw::button.outline>
+    <x-rapidez-mw::button.accent @click="addWishlist('{{ __('New wishlist') }}')">
```

Comment:
Since we're mixing blade & vue components my personal preference goes towards using the `v-on:` syntax
```suggestion
    <x-rapidez-mw::button.accent v-on:click="addWishlist('{{ __('New wishlist') }}')">
```
Functionally nothing changes because we do not use Livewire, but it's easier to read when working with Vue & Livewire when it is explicit